met-cron/src/met_cron/services/user_etl_service.py

```diff
@@ -0,0 +1,65 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Service to do ETL on comments."""
+
+from datetime import datetime, timedelta
+from typing import List
+
+from flask import current_app
+from met_api.models.comment import Comment as MetCommentModel
+from met_api.constants.comment_status import Status as CommentStatus
+from met_cron.models.db import db
+from met_api.models.user import User as MetUserModel
+from met_cron.models.user_details import UserDetails as UserDetailsModel
+
+
+class UserEtlService:  # pylint: disable=too-few-public-methods
+    """ETL Service on users."""
+
+    @staticmethod
+    def do_etl_users():
+        """Perform the ETL for users."""
+        current_app.logger.info('<<<<<<<<<<<<<<<<<<<<<<<<<<<User Extraction starting ')
+        updated_users = UserEtlService._get_updated_users()
+        if not updated_users:
+            current_app.logger.info('No new Users Found')
+            return
+        current_app.logger.info('Total updated Users Found : %s.', len(updated_users))
+        UserEtlService._transform_users(updated_users)
```

Comment:
hmm, I noticed this etl method and another only call transform and the transform method calls load. transform method should only transform. ETL method is better to call transform get the result then with the  result call load. Would you disagree?

Reply:
yup..valid point..
I changed the method to load_users